analyzers/tests/SonarAnalyzer.UnitTest/TestCases/UninvokedEventDeclaration.cs

```diff
@@ -204,5 +204,23 @@ public void InvokeAll()
                 outer.MyAction6?.Invoke(this, EventArgs.Empty);
             }
         }
+
+        // https://github.com/SonarSource/sonar-dotnet/issues/3931
+        public delegate void SomeDelegate(int a);
+
+        public class ReproForIssue3931
+        {
+            public event SomeDelegate ActuallyUsedEvent; // Noncompliant FP
```

Comment:
I'm not sure I get it. Where is `ActuallyUsedEvent` used?

Reply:
I messed it up, I`ll fix it. I tried to combine it with the example from the community and messed it up. Fix is on its way. Sorry...

Reply:
No worries, it looks good now. 

I would add though an additional case also with a call of `.GetInvocationList();` on another event for which we don't do an iteration. We will have to decide if a call to `.GetInvocationList()` will "sanitize" the event or if we want to actually check for iterations and invoke calls.